overrides/partials/header.html

```diff
@@ -0,0 +1,70 @@
+{#-
+    This file was automatically generated - do not edit
```

Comment:
Can we remove this comment?

Reply:
We can, its better to keep it as it is, to keep  in track with the partial override from the material's template, when theme is updated